fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -285,6 +297,29 @@ const CreateElection = () => {
         {STRINGS.election_create_add_question}
       </PoPTextButton>
 
+      {!isConnected && (
+        <Text style={[Typography.paragraph, Typography.error]}>
+          {STRINGS.event_creation_must_be_connected}
+        </Text>
+      )}
+      {electionName === '' && (
+        <Text style={[Typography.paragraph, Typography.error]}>
+          {STRINGS.event_creation_name_not_empty}
+        </Text>
+      )}
+      {questions.some(isQuestionInvalid) && (
+        <Text style={[Typography.paragraph, Typography.error]}>
+          {STRINGS.election_create_invalid_questions_1 +
+            MIN_BALLOT_OPTIONS +
+            STRINGS.election_create_invalid_questions_2}
```

Comment:
This seems like a slightly fragile approach to handling a variable in a translation.
It's generally safer to have the full text in one string, with a variable that gets replaced here.
e.g. `myStr = "there are {} elephants"` , and `myStr.replace("{}",3)`, because then it becomes easy to change the text to say `"{} elephants were seen"` , or translate to a different language where `{}` would be in a different place in the string.